<launch>
  <node name="position_controller" pkg="drive_controller" type="drive_controller_master">
    <param name="simulating_driving" type="bool" value="true"/>
    <param name="localization_settle_time" type="double" value="5"/>
  </node>
</launch>
